File: de.carsten_lenz.liquibasegui/src/de/carsten_lenz/liquibasegui/ui/OpenConsoleViewHandler.scala
package de.carsten_lenz.liquibasegui.ui

import org.eclipse.core.commands._
import org.eclipse.ui.console._

class OpenConsoleViewHandler extends AbstractHandler with HandlerHelpers {

    def execute(e: ExecutionEvent): Object = {
        implicit val event = e; // enables implicits on HandlerHelpers
        
        val messageConsole = new MessageConsole("Test", null)
        ConsolePlugin.getDefault().getConsoleManager().addConsoles(Array[IConsole](messageConsole))
        
        val stream = messageConsole.newMessageStream
        
        stream.println("Dies ist ein Konsolentest")
        
        null
    }
}

File: de.carsten_lenz.liquibasegui/src/de/carsten_lenz/liquibasegui/ui/OpenConsoleViewHandler.java
package de.carsten_lenz.liquibasegui.ui;

import org.eclipse.core.commands.AbstractHandler;
import org.eclipse.core.commands.ExecutionEvent;
import org.eclipse.ui.console.ConsolePlugin;
import org.eclipse.ui.console.IConsole;
import org.eclipse.ui.console.MessageConsole;
import org.eclipse.ui.console.MessageConsoleStream;


class OpenConsoleViewHandler extends AbstractHandler {

    public Object execute(ExecutionEvent event) {
        MessageConsole messageConsole = new MessageConsole("Test", null);
        ConsolePlugin.getDefault().getConsoleManager().addConsoles(new IConsole[]{ messageConsole});
        
        MessageConsoleStream stream = messageConsole.newMessageStream();
        
        stream.println("Dies ist ein Konsolentest");
        
        return null;
    }
}